charts/events/templates/deployment.yaml

```diff
@@ -55,6 +67,7 @@ spec:
             - autoscaling/v1
```

Comment:
What happens when either of these three are specified in `-exclude-target` or `-include-target`?

Reply:
Adding `-include-target` doesn't add any value. I included it for possible future use. Do you think we should drop it?

I did catch an error, should be [] and not {}. I'll update accordingly, but would appreciated thoughts on omitting `includeTargets`.

Reply:
I think it's good to have that, but my question was different. It looks like additional APIs can be included by simply specifying `-<API>`, which you can see specified as `autoscaling/v1`, `batch/v1`, `networking.k8s.io/v1`. I was curious as to how the new flags behave with these in tandem.